fdbserver/networktest.actor.cpp

```diff
@@ -24,6 +24,44 @@
 
 UID WLTOKEN_NETWORKTEST( -1, 2 );
 
+struct latency_stats {
```

Comment:
I think most of our codebase does CamelCaseNames, so give it a `:%s/latency_stats/LatencyStats/`.  This applies to the helper functions below as well.  For variable names, most code does whateverCaseThisIsCalled, but some do the snake_case, so generally just try to fit in with whatever style you're around?